packages/common/src/store/purchase-content/sagas.ts

```diff
@@ -11,7 +11,14 @@ import {
   getTokenAccountInfo,
   purchaseContent
 } from 'services/audius-backend/solana'
+import { FeatureFlags } from 'services/index'
```

Comment:
Okay seeing as this is the third time I've mentioned it, I feel compelled to suggest we figure out why the tooling is preferring an import from such a high point that has to process the whole import tree under it instead of importing directly from the appropriate submodule.

Reply:
I think it's probably sorting by tree depth then by alphabetical:
![image](https://github.com/AudiusProject/audius-protocol/assets/3690498/29341330-1361-4635-90f6-3b6658b7640d)

Reply:
![image](https://github.com/AudiusProject/audius-protocol/assets/3690498/967b1550-48ff-41e9-9358-57d3dcda8419)
though I might be wrong!